### PR TITLE
Java coding exam 4

### PR DESCRIPTION
**Pull request by NagendraTOct2020**

### CODE REVIEW

File: src/nagendra/CodingExam4.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+public class CodingExam4 {
+
+	
+	void getCommonElement(int[] input,int[] input2) {
+		for(int i=0;i<input.length;i++){
+			for(int j=0;j<input2.length;j++){
+				if(input[i]==input2[j]){
+					System.out.println("Common :"+input[i]);
```
Comment (harshvegada) [inline src/nagendra/CodingExam4.java:18]: Just need to put break, once you get element value then you can stop iteration on second array,

File: src/nagendra/CodingExam4A.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+*/
+
+public class CodingExam4A {
+	
+	void getIdenticalString(String userInput, String input) {
+		boolean flag = true;
+		char[] input1=userInput.toLowerCase().toCharArray();
+		char[] input2=input.toLowerCase().toCharArray();
+       for(int i=0;i<input1.length;i++){
+    	   for(int j=i;j<=i;j++){
```
Comment (harshvegada) [inline src/nagendra/CodingExam4A.java:21]: you can handle in single loop,
and need to use charAt() method to compare char value, if not match then update flag value and break the loop,

**Review by harshvegada — CHANGES_REQUESTED** [review]
Work on comments for 2nd String program

File: src/nagendra/CodingExam4.java (hunk 1 above)
Comment (NagendraTOct2020) [reply src/nagendra/CodingExam4.java:17]: Updated as per comment

Comment (harshvegada) [reply src/nagendra/CodingExam4.java:17]: Thanks Nagendra

File: src/nagendra/CodingExam4A.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+public class CodingExam4A {
+
+	void getIdenticalString(String userInput, String input) {
+		boolean flag = true;
+		String input1 = userInput.toLowerCase();
+		String input2 = input.toLowerCase();
+		for (int i = 0; i < input1.length(); i++) {
+			if (input1.charAt(i) != input2.charAt(i))
+				flag = false;
```
Comment (harshvegada) [inline src/nagendra/CodingExam4A.java:22]: here also need to put break;

**Review by harshvegada — APPROVED** [review]
Good to merge!

**Pull request merged**